---
title: C6322
ms.date: 11/04/2016
ms.prod: visual-studio-dev15
ms.technology: vs-ide-code-analysis
ms.topic: reference
f1_keywords:
- C6322
helpviewer_keywords:
- C6322
ms.assetid: fb23d2b1-b2a0-465c-8bf5-ec039c6c7757
author: mikeblome
ms.author: mblome
manager: wpickett
ms.workload:
- multiple
ms.openlocfilehash: 4ae06070a2cd266f4a87b6c4ff287ab4ad423939
ms.sourcegitcommit: e13e61ddea6032a8282abe16131d9e136a927984
ms.translationtype: MT
ms.contentlocale: es-ES
ms.lasthandoff: 04/26/2018
ms.locfileid: "31893332"
---
# <a name="c6322"></a>C6322
Advertencia C6322: Bloque empty _except

 Este mensaje indica que no hay ningún código en el bloque _except. Como resultado, las excepciones podrían no controlarse.

## <a name="example"></a>Ejemplo
 El código siguiente genera esta advertencia:

```
#include <stdio.h>
#include <excpt.h>
#include <windows.h>

void fd(char *pch)
{
   __try
     {
       // exception rasied if pch is null
       *pch= 0 ;
     }
   __except(GetExceptionCode() == EXCEPTION_ACCESS_VIOLATION)
     {
     }
}
```

 Para corregir esta advertencia, utilice el código siguiente:

```
#include <stdio.h>
#include <excpt.h>
#include <windows.h>

void f(char *pch)
{
   __try
     {
       // exception rasied if pch is null
      *pch= 0 ;
     }
   __except(GetExceptionCode() == EXCEPTION_ACCESS_VIOLATION ?
               EXCEPTION_EXECUTE_HANDLER : EXCEPTION_CONTINUE_SEARCH)
     {
       // code to handle exception
       puts("Exception Occurred");
     }
}
```

## <a name="see-also"></a>Vea también
 [try-except (Instrucción)](/cpp/cpp/try-except-statement)